src/js/navigation.js

```diff
@@ -22,9 +22,15 @@ function insertUserNavItem (user) {
 	return ({ selectorContainer, selectorInsertionPoint, selectorLink }) => {
 		const elCt = $(selectorContainer);
 
-		const elInsertionPoint = elCt.querySelector(selectorInsertionPoint).parentElement;
+		const navLink = elCt.querySelector(selectorInsertionPoint);
+		if (!navLink)
+			return;
```

Comment:
can this be wrapped in `{}` please for consistencies sake

Reply:
Done.
There's no tests on this code but I verified it by symlinking from n-ui and running the demo page.